test/test-api-getreport.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-// Testcase for returning NodeReport as a string via API call
+// Testcase for returning report as a string via API call
 
 const common = require('./common.js');
 const tap = require('tap');
 const nodereport = require('../');
 var report_str = nodereport.getReport();
 common.validateContent(report_str, tap, {pid: process.pid,
-  commandline: [process.argv0, 'test/test-api-getreport.js'].join(' ')
-});
\ No newline at end of file
+  commandline: [process.argv[0], 'test/test-api-getreport.js'].join(' ')
```

Comment:
I'm also not sure how https://ci.nodejs.org/view/post-mortem/job/nodereport-continuous-integration/52/MACHINE=win10/console passed given that the slash here is incorrect for Windows (test is failing locally for me).